frontend/src/components/Collections/components/Grid/components/Row/CollectionRow/index.tsx

```diff
@@ -24,7 +23,7 @@ interface Props {
   id: string;
   accessType?: ACCESS_TYPE;
   visibility: VISIBILITY_TYPE;
-  revisionStatus?: REVISION_STATUS;
+  revisionsEnabled?: boolean;
```

Comment:
The changes on this file are just swapping out the old `REVISION_STATUS` for the new collection attribute